app/tools/extractRegisterForReflection.java

```diff
@@ -0,0 +1,25 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.reflections:reflections:0.10.2
+//DEPS io.apicurio:apicurio-data-models:1.1.26
+
+import java.util.*;
```

Comment:
I like to avoid starts for imports, but it's just a personal thing.

Reply:
fixed!